implementations/rust/ockam/ockam_node/src/router/stop_worker.rs

```diff
@@ -9,10 +9,12 @@ use ockam_core::{Address, Result};
 pub(super) async fn exec(
     router: &mut Router,
     addr: &Address,
+    bare: bool,
```

Comment:
Hm... Any other options for the name? Detached? Autonomous?

Reply:
I created a thread on slack to find a better name. I do agree that "bare context" is very ambiguous :sweat_smile: